src/components/groups/index.js

```diff
@@ -4,6 +4,7 @@ import * as usersService from "../../services/users-service";
 import * as authService from "../../services/auth-service"
 import GroupsList from "./group-list";
 import {useNavigate} from "react-router-dom";
+import { Link } from "react-router-dom";
```

Comment:
Good work. Could probably be combined with the useNavigate import since they both come from react-router-dom

Reply:
Agree^ still works though and looks great. Send it home